includes/widgets/accordion.php

```diff
@@ -114,6 +126,19 @@ protected function register_controls() {
 			]
 		);
 
+		$is_nested_tabs_active = Plugin::$instance->widgets_manager->get_widget_types( 'nested-accordion' );
```

Comment:
You don't really need to cache this in a variable since it's only used once. But you can decide :-)